Sources/Hub/HubApi.swift

```diff
@@ -106,6 +106,13 @@ public extension HubApi {
     /// `fileURL` is a complete local file path for the given model
     func configuration(fileURL: URL) throws -> Config {
         let data = try Data(contentsOf: fileURL)
+        do {
+            let parsed = try JSONSerialization.jsonObject(with: data, options: [])
+            guard let dictionary = parsed as? [String: Any] else { throw Hub.HubClientError.parse }
```

Comment:
```suggestion
            guard let dictionary = parsed as? [NSString: Any] else { throw Hub.HubClientError.parse }
```

My bad, made a mistake in the merge